hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -756,6 +814,26 @@ public boolean isGrpcBlockTokenEnabled() {
     return grpcBlockTokenEnabled;
   }
 
+  /**
+   * Return config value of {@link OMConfigKeys#OZONE_OM_MULTITENANCY_ENABLED}.
+   */
+  public boolean isS3MultiTenancyEnabled() {
```

Comment:
I don't see any callers of this method. If we make config errors fatal it would be the same as checking the value of the config key.

Reply:
Updated javadoc. The intention is to have a method to "safely" check if it is enabled just like other configs. And yes it is unused for now.